src/styles/blocks/card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
```

Comment:
For consistency with the code style, the margin-bottom value should also be a multiple of 4. Consider adjusting it from `40px` to `32px` or `44px`.